README_BUILD.md

```diff
@@ -135,17 +136,47 @@ The `install_sdk`command will install SDKs (build tools etc) such as the Android
 
 If you wish to build for any other platform, you will need to install an sdk package where the build system can find it.
 
-Next thing you need to do is to install external packages:
+There are two optons how to host prebuilt packages:
+1. Store at local machine
+2. Store in S3-compatable object storage (custom Minio server, Hetzner object storage, AWS S3, etc.)
 
-```sh
-$ ./scripts/build.py install_sdk --platform=... --package-path=...
-```
+To control access to location where prebuilt packages stored there are following command-line arguments/environment variables:
+| Argument                 | Environment variable           | Description                                                                                                                              | Example                                                                                                               |
+|--------------------------|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| --package-base-url       | **DM_PACKAGES_BASE_URL**       | Base S3-compatable object storage URL or absolute path to local folder                                                                   | Object storage: DM_PACKAGES_BASE_URL=play.min.io. Local folder: DM_PACKAGES_BASE_URL=/Users/user/work/defold-packages |
+| --package-bucket         | **DM_PACKAGES_BUCKET**         | Bucket name in S3 object storage where prebuilt packages stored                                                                          | DM_PACKAGES_BUCKET=prebuilt-packages                                                                                  |
+| --package-access-key     | **DM_PACKAGES_ACCESS_KEY**     | S3 object storage access key id. Account with such Key ID should be allowed to perform `s3:Getobject` and `s3:GetBucketLocation` actions |                                                                                                                       |
+| --package-access-secret  | **DM_PACKAGES_ACCESS_SECRET**  | S3 object storage access secret                                                                                                          |                                                                                                                       |
+| --package-encryption-key | **DM_PACKAGES_ENCRYPTION_KEY** | Hex presentation of 32 bytes long encryption key used when files were uploaded to object storage                                         |                                                                                                                       |
 
-You could also set the package path in an environment variable `DM_PACKAGES_URL`:
+Next thing you need to do is to install external packages:
```

Comment:
In order to ease our burden of explaining this to newcomers, the `need` is too strict.
We should emphasize that on all platforms, we support _local sdks_.
The `install_sdk` is for advanced use only.